charts/kube-prometheus-stack/values.yaml

```diff
@@ -2222,6 +2222,17 @@ prometheusOperator:
     tag: v0.30.2
     sha: ""
 
+  ## Set a Label Selector to filter watched prometheus and prometheusAgent
+  ##
+  promSelector: ""
```

Comment:
Can you name this prometheusInstanceSelector and add Instance to the other 2?

Reply:
> Can you name this prometheusInstanceSelector and add Instance to the other 2?

Done